docs/content/en/_index.html

```diff
@@ -12,13 +12,20 @@
 </div>
 
 <div class="mx-auto">
-	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
-		Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs/quickstart/" >}}">
+		Quick Start <span class="ml-2">🚀</span>
```

Comment:
To be honest, I don't really like this rocket icon much. Its color does not match our theme.
No strong opinion, but one vote from me to remove it.

Reply:
okay 😂  then what should I use there, should we keep the arrow as the existing version of the cover 👀

Reply:
@cakecatz Is there any good way to add a script to load the latest version from this file when the page is loaded?
https://github.com/pipe-cd/pipe/blob/master/release/RELEASE

Reply:
Reverted to use the previous arrow icon 🙏